docs/ios-http-instrumentation.md

```diff
@@ -102,3 +146,30 @@ let task = URLSession.shared.dataTask(with: urlRequest) { data,response,error  i
   ...
 }
 ```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## 
+
+For Alamofire users you can also use a helper class that implements `RequestAdapter` protocol, and using this class session instead of Alamofire default one.
```

Comment:
```suggestion
If you use Alamofire, you can also use a helper class that implements the `RequestAdapter` protocol, and use this class session instead of Alamofire's default one.
```